lib/utils.js

```diff
@@ -1,6 +1,22 @@
 // Custom Splitter:
-const split = (str, delimiter) => {
-    return (str.split(delimiter || ',').map(element => element.trim())).filter(element => element);
+const split = (str, delimiter, haswhitespace) => {
+    return str.split(delimiter || ',').map(element => {
+        if (haswhitespace) {
+            element = element.replace(/^"+/, '').replace(/"+$/, '');
+            element = element.replace(/^'+/, '').replace(/'+$/, '');
+            element = element.replace(/\+/, ' ');
+        }
+        return element.trim();
+    }).filter(element => element);
+};
+
+// Prepare Message Reporting the Options Affected:
+const preparePostMessage = (diff, length) => {
```

Comment:
I'd just pass in `diff` as the sole param and use `diff.length` where you need. You already do that in line 17 and it's a bit odd to mix that with also using a `length` param.

Reply:
Oh, I forgot to remove - gonna remove the `diff.length` from line 17 - I wanna pass `length` since I use it in multiple places - don't wanna define new variable inside function - just calculating once and passing into the function

Reply:
I'd actually much prefer the opposite - only pass in `diff` and use `diff.length` everywhere. If you *really* want (although I think it's overkill), you could do `const length = diff.length` at the top of this fn. But it's error prone to depend on callers to pass things with the right state (here that `length` reflects `diff` and here it's unnecessary.

Reply:
I see - you are right - just updated!